plugins/inputs/tail/tail.go

```diff
@@ -87,23 +92,25 @@ func (t *Tail) Start(acc telegraf.Accumulator) error {
 	t.acc = acc
 	t.tailers = make(map[string]*tail.Tail)
 
-	return t.tailNewFiles(t.FromBeginning)
+	err := t.tailNewFiles(t.FromBeginning)
+
+	// clear resume offsets after starting
+	offsetsMutex.Lock()
+	offsets = make(map[string]int64)
+	offsetsMutex.Unlock()
```

Comment:
If there are more than one Tail plugin, only one of them might have it's offsets restored.  There isn't a great spot to put this, but perhaps we can copy the offsets in the `Init()` function for now, it is called for all plugins before `Start()`.

Reply:
Oh damn I hadn't thought about that. To solve - I am storing a snapshot of `offsets` in the Tail struct, copying during `NewTail()`. Reasonable?

Reply:
Yes that should work well.